Stack/Opc.Ua.Core/Stack/Configuration/ApplicationConfiguration.cs

```diff
@@ -146,33 +142,11 @@ public IList<string> GetServerDomainNames()
                 }
             }
 
-            for (int ii = 0; ii < baseAddresses.Count; ii++)
-            {
-                Uri url = Utils.ParseUri(baseAddresses[ii]);
-
-                if (url == null)
-                {
-                    continue;
-                }
-
-                string domainName = url.DnsSafeHost;
-
-                if (url.HostNameType == UriHostNameType.Dns)
-                {
-                    domainName = Utils.ReplaceLocalhost(domainName);
-                }
-                else // IPv4/IPv6 address
-                {
-                    domainName = Utils.NormalizedIPAddress(domainName);
-                }
-
-                if (!Utils.FindStringIgnoreCase(domainNames, domainName))
-                {
-                    domainNames.Add(domainName);
-                }
-            }
-
-            return domainNames;
+            var baseUrls = baseAddresses.Where(a => !String.IsNullOrEmpty(a)).Select(a => Utils.ParseUri(a));
```

Comment:
I would prefer to not change this block.
I *think* it will work but I don't see how distinct handles the ignore case (does it need an IEquality comparer with ordinalignorecase?). Apparently we don't have tests yet so better no risk here. Thanks!

Reply:
@mregen I haven't looked at it yet but would you prefer tests if it's something I can do to prove it works and if not just change it back?
I've signed it a few times but it still is saying I haven't so I'm not sure what is going on there.